fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -648,6 +648,31 @@
 | `aria-=(null)`| (initial, warning)| `<null>` |
 | `aria-=(undefined)`| (initial, warning)| `<null>` |
 
+## `aria-hidden` (on `<div>` inside `<div>`)
```

Comment:
This shows why you should currently use `<div aria-hidden={ariaHidden ? true : undefined} />` since `aria-hidden="false"` is currently not recommended due to browser inconsistencies.